vtex/checkout-ui-tests
# Cypress update

#### What is the purpose of this pull request?
Improve reliability of the tests, by updating cypress and fixing some scenarios.

#### What problem is this solving?
Some tests are failing due to platform instability, configuration and other problems. 

#### How should this be manually tested?
Running a new test group locally should be sufficient (check project documentation, but basically run command below)

`yarn cypress run --spec "tests/payment/*Expired*"` 

#### Screenshots or example usage
`yarn cypress run --spec "tests/payment/*Expired*"`

![image](https://user-images.githubusercontent.com/5751468/112388190-8dcce100-8cd1-11eb-92f2-cdc3cf89ae2c.png)

#### Types of changes
